---
layout: post
title: Real-time genomic epidemiology of nCoV-2019 (Wuhan novel coronavirus)
description: place holder
image: assets/images/wuhan.jpg
show_tile: false
permalink: ncov-2019
category: menu
---

There is a pressing need to understand more about the short-term genomic epidemiology and evolution of the recently described novel coronavirus (nCoV-2019). Initial cases were in Wuhan City, Hubei Province, China but now cases have been confirmed both more widely in China and internationally. 

Viral genome data generated prospectively during outbreaks can help provide information about relatedness to other viruses, mode and tempo of evolution, geographical spread and adaptation to human hosts. This information can be used to assist in epidemiological investigations, particularly when combined with other types of data (e.g. case counts).

The ARTIC network is making available a set of materials (see below) to assist groups in sequencing the virus including a set of primers, laboratory protocols, bioinformatics tutorials and datasets. These are mainly focused around the use of the portable Oxford Nanopore MinION sequencer, although aspects of the protocol such as the primer scheme and sample amplification may be generalised to other sequencing platforms.

The protocol relies on direct amplification of the virus using tiled, multiplexed primers. This approach has been proven to have high sensitivity and work directly from clinical samples (compared to metagenomics approaches). This protocol is currently unvalidated, we will add validation data as we generate it to this page. However, the techniques described have been extensively validated in other RNA viruses including Ebola, Zika and yellow fever.

The rapid development of this primer scheme is possible as the complete genome sequences for a number of cases were released shortly after the identification of the virus both on GenBank ([accession MN908947](https://www.ncbi.nlm.nih.gov/nuccore/MN908947)) and on [GISAID](http://gisaid.org). The acknowledgments for the labs and people involved in generating these sequences is given below and we wish to highlight the cooperative spirit in which this data was shared. We urge anyone who uses this protocol to share the resulting genomes to benefit the international response to this epidemic.

Please subscribe to our <a href="https://groups.google.com/forum/#!forum/ncov-2019-sequencing">Google Group</a> to receive email updates regarding these protocols, or follow us on <a href="https://twitter.com/NetworkArtic">Twitter (@NetworkArtic)</a>.

## Resources and documents

{% assign docs = site.resources | where_exp:"item", "item.category contains 'ncov'" | sort: 'title' %}
<ul>
    <li>nCoV-2019 sequencing protocol (protocols.io)</li>
        <blockquote>link: <a href="https://dx.doi.org/10.17504/protocols.io.bbmuik6w">Protocols.io</a></blockquote>

    <li>nCoV-2019 PrimalSeq sequencing primers</li>
        <blockquote>link: <a href="https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCov-2019/V1">Github</a></blockquote>

{% for doc in docs %}
    <li>{{ doc.title_text }}</li>
	<blockquote>link: <a href="{{ doc.permalink }}">{{ doc.permalink }}</a></blockquote>
{% endfor %}
</ul>


## Further Reading

1. Rambaut A, Preliminary phylogenetic analysis of 11 nCoV2019 genomes, 2020-01-19 <http://virological.org/t/preliminary-phylogenetic-analysis-of-11-ncov2019-genomes-2020-01-19/329>

2. WHO's code of conduct for open and timely sharing of pathogen genetic sequence data during outbreaks of infectious disease, <https://www.who.int/blueprint/what/norms-standards/GSDDraftCodeConduct_forpublicconsultation-v1.pdf?ua=1>

## Data acknowledgements

These resources were only possible because of the genome sequence ([accession MN908947](https://www.ncbi.nlm.nih.gov/nuccore/MN908947)) released by The Shanghai Public Health Clinical Center & School of Public Health, in collaboration with the Central Hospital of Wuhan, Huazhong University of Science and Technology, the Wuhan Center for Disease Control and Prevention, the National Institute for Communicable Disease Control and Prevention, Chinese Center for Disease Control, and the University of Sydney, Sydney, Australia.

We would also like to acknowledge all the labs that shared genome sequences through the GISAID plaform. Although these data were not used directly, they confirm the genome sequence and demonstrate a current lack of diversity allowing a more sensitive sequencing protocol.
