---
title: "MPXV sequencing | A list of resources for sequencing and bioinformatics"
keywords: protocol
layout: document
last_updated: 2024-12-12
tags: [protocol] 
permalink: /mpxv/resources.html
folder: mpxv
title_text: "Mpox virus (MPXV) resources"
subtitle_text: "A list of resources for sequencing and bioinformatics"
document_name: "ARTIC-MPXV-resources"
version: v1.0.0
creation_date: 2024-08-23
forked_from: 
author: ARTIC team
citation: "Loman *et al.* In Prep."
nav_menu: false
show_tile: false
category: mpxv-guide
---

{% include callout.html
type='default'
content='**Overview:** This document is part of the Mpox virus (MPXV) sequencing protocol package: [http://artic.network/mpxv](http://artic.network/mpxv)'
%}

## Background

> [Mpox virus sequencing | A guide to sequencing for genomic epidemiology](/mpxv/artic-mpxv-guide)
: This is an overview guide and set of recommendations for MPXV genome sequencing with an emphasis on genomic epidemiology. 

## Setup Guides

> [Setting up and using EPI2ME for MPXV analysis pipelines](/mpxv/mpxv-epi2me-setup.html) 
: This document walks-through how to install and setup Oxford Nanopore Technology's EPI2ME software package so that it can be used to run ARTIC bioinformatics pipelines on a Desktop or Laptop computer without having to use a command-line interface.

## MPXV sequencing

> ARTIC/INRB Clade-independent Primer Schemes
: 2500 bp primer scheme [Clade-I reference](https://labs.primalscheme.com/detail/artic-inrb-mpox/2500/v1.0.0-cladeia/?q=mpox), [Clade II reference](https://labs.primalscheme.com/detail/artic-inrb-mpox/2500/v1.0.0-cladeib/?q=mpox), [/mpxv/mpox-multiplex-pcr-protocol-2500.docx](PCR amplification protocol)
: 400 bp primer scheme [Clade-I reference](https://labs.primalscheme.com/detail/artic-inrb-mpox/400/v1.0.0-cladeia/?q=mpox), [Clade II reference](https://labs.primalscheme.com/detail/artic-inrb-mpox/400/v1.0.0-cladeib/?q=mpox), [/mpxv/mpox-multiplex-pcr-protocol-400.docx](PCR amplification protocol)

> Tiled amplicon sequencing protocol and primer scheme for MPXV
: Chen *et al.* Yale amplicon sequencing protocol \- [dx.doi.org/10.17504/protocols.io.5qpvob1nbl4o/v4](http://dx.doi.org/10.17504/protocols.io.5qpvob1nbl4o/v4)
: This protocol is a primer scheme and validation for MPXV of the [ARTIC SARS-CoV-2 tiled amplicon protocol](https://dx.doi.org/10.17504/protocols.io.bp2l6n26rgqe/v4)

## Bioinformatics pipeplines
        
> [Running the ARTIC nanopore bioinformatics pipeline using EPI2ME](/mpxv/mpxv-ont-epi2me-sop.html)
: This document walks-through how to install and setup Oxford Nanopore Technology's EPI2ME software package so that it can be used to run ARTIC bioinformatics pipelines on a Desktop or Laptop computer without having to use a command-line interface.

> [Running a bioinformatics pipeline for Illumina data using EPI2ME](/mpxv/mpxv-illumina-epi2me-sop.html)
: This document walks-through how to install and setup Oxford Nanopore Technology's EPI2ME software package so that it can be used to run ARTIC bioinformatics pipelines on a Desktop or Laptop computer without having to use a command-line interface.

> [Running the ARTIC nanopore bioinformatics pipeline using the command line](/mpxv/mpxv-bioinformatics-sop.html)
: This document provides detailed instructions for running the ARTIC fieldbioinformatics pipeline for MPXV on the command-line. This is useful to review what the pipeline does and how it works and for debugging issues for particular datasets. 

## Analysis pipeline

> [Running the Squirrel MPXV alignment and phylogenetics pipeline using EPI2ME](/mpxv/mpxv-phylogenetics-epi2me-sop.html)
: This document walks-through how to install and setup Oxford Nanopore Technology's EPI2ME software package so that it can be used to run ARTIC bioinformatics pipelines on a Desktop or Laptop computer without having to use a command-line interface.

**Further Reading / Viewing**

EPI2ME Labs \- [https://labs.epi2me.io/](https://labs.epi2me.io/)

ARTIC-CLIMB workshop videos (covering SARS-CoV-2 but generally applicable to MPXV) \- [https://www.climb.ac.uk/artic-and-climb-big-data-joint-workshop-2/](https://www.climb.ac.uk/artic-and-climb-big-data-joint-workshop-2/)

Chen *et al.* Yale amplicon sequencing protocol \- [https://www.protocols.io/view/monkeypox-virus-multiplexed-pcr-amplicon-sequencin-5qpvob1nbl4o](https://www.protocols.io/view/monkeypox-virus-multiplexed-pcr-amplicon-sequencin-5qpvob1nbl4o)

Squirrel \- *[https://github.com/aineniamh/squirrel*](https://github.com/aineniamh/squirrel)

 ---

{% include callout.html
type='default'
content="This document is part of the MPXV sequencing protocol package: [http://artic.network/mpxv](http://artic.network/mpxv)"
%}

<br /><br />

{% include wellcome-trust.html %}
<br />

<div class="pagebreak"> </div>

