---
title: "MPXV alignment and phylogenetics protocol | Squirrel"
keywords: protocol
layout: document
last_updated: 2024-08-16
tags: [protocol]
summary:
permalink: /mpxv/mpxv-phylogenetics-epi2me-sop.html
folder: mpxv
title_text: "MPXV alignment and phylogenetics pipeline using Epi2Me"
subtitle_text: "Squirrel | bioinformatics"
document_name: "ARTIC-MPXV-phylogeneticsSOP"
version: v1.0
creation_date: 2024-08-21
revision_date: 
forked_from: 
author: √Åine O'Toole
citation: https://github.com/aineniamh/squirrel
nav_menu: false
show_tile: false
category: mpxv-epi2me
---

{% include callout.html
type='default'
content='**Overview:** A complete protocol to take the output consensus genome sequences from the [sequencing protocol](/mpxv/mpxv-seq-sop.html) to a robust, interpretable phylogenetic tree with APOBEC3-editing reconstruction. Includes background dataset, alignment, maximum likelihood phylogenetics and ancestral reconstruction with IQTREE, figure generation and interpretation.
'
%}

### Rationale
MPXV is a large poxvirus, with a complex dsDNA genome ~200kb in length. Alignment of sequences, and therefore phylogenetics, is challenging using classic due to tracts of low-complexity and repetitive regions. Squirrel provides an efficient map-to-reference alignment pipeline with masking of problematic regions of the genome.

### Command line
Squirrel can be used as a command line tool, with full command-line documentation available on the squirrel GitHub repository at (github.com/aineniamh/squirrel)[https://github.com/aineniamh/squirrel].

### User interface
Squirrel can also be run through the [EPI2ME](https://labs.epi2me.io/downloads/) user interface. Please first install the EPI2ME desktop application using the provided link. You can then go to 'available workflows' then 'Import workflow' from [`https://github.com/artic-network/squirrel-nf`](https://github.com/artic-network/squirrel-nf) as shown below:

<img width="500" alt="link" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_1.png">

Once the workflow has successfully downloaded, you can click the X to exit to download window, and select it from the list of available workflows. Next select `Run this workflow` from the available options, and then `Run on your computer`:

<img width="500" alt="launch" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_2.png">
<img width="500" alt="run" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_3.png">

This will bring up a menu where you can provide the inputs for your analysis. The only required file is a single FASTA file containing all the sequences and outgroups for your analysis and you must also select the clade (i or ii) from the drop down list:

<img width="500" alt="fasta" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_4.png">
<img width="500" alt="clade" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_5.png">

Running with just a FASTA file will generate an alignment of the input sequences. We recommend selecting the check box for `Seq QC` to check this alignment for problematic sites. 

<img width="500" alt="seqqc" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_6.png">

Scrolling down the menu, select the box to `Run Phylo`. At this point you have 2 options. EITHER you can select the check box to `Include Background`, in which case a default panel of clade-specific outgroups sequences will be used.

<img width="500" alt="includebackground" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_7.png">

OR you can specify a number of outgroups IDs. These outgroups must be present in the FASTA file you provided and will be pruned out of the final alignment. For Clade I we recommend outgroups KJ642617,KJ642615,KJ642616 and for Clade IIb we recommend KJ642617,KJ642615. If you also selected the `Include Background` option your specified outgroups will be ignored.

<img width="500" alt="outgroups" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_8.png">

Optionally you can provide a different reference sequence, but this is usually unnecessary - a clade specific reference will be used by default. No Advanced Options or Nextflow Configuration options are required by default.

Click Launch: 

<img width="500" alt="launch2" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_9.png">

This will start the workflow. A progress bar is displayed with the run status but you will not be able to see the stdout that is generated on the command line. 

<img width="500" alt="progress" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_10.png">

Once the run is completed, a number of files will be available and you can double-click to view them:

<img width="500" alt="complete" src="/assets/images/mpxv/phylogenetics-sop/screen_shot_9.png">

This includes a suggested_mask.csv file generated by the run with potentially problematic sites. If you start a new run with the same inputs and additionally provide this mask file in the menu, it will improve the alignment and phylogeny.


#### Related documents:
This document is part of the MPXV sequencing protocol package:
: [http://artic.network/mpxv/](http://artic.network/mpxv/)

Setting up the laptop computing environment using Conda:
: [http://artic.network/ebov/ebov-it-setup.html](http://artic.network/ebov/ebov-it-setup.html)



<br /><br /><br />

{% include wellcome-trust.html %}

<div class="pagebreak"> </div>
